client/src/services/injectElipsis.js

```diff
@@ -0,0 +1,13 @@
+const injectElipsis = (html) => {
+    const re = /<\/p>/g
+    const str = html;
+    let lastMatchIndex;
+    let match;
+    while ((match = re.exec(str)) != null) {
+        lastMatchIndex = match.index;
+    }
+
+    return html.substring(0, lastMatchIndex) + "..." + html.substring(lastMatchIndex)
+}
+
+export default injectElipsis;
```

Comment:
I think there is a typo in the name `injectElipsis` which should be `injectEllipsis`. Kindly change the file name, the variable name, export name, and wherever imported.